### DISCUSSION TITLE
How to update the stock faster?

### DISCUSSION THREAD

**Discussion by icredhat** [jef/streetmerchant#1488] [Help] [answered]
Right now it looks up for stock every 4-5 seconds , but i would like to set it up so it updates every 1-2 seconds how can i do that?
I have already changed in the config.ts
MaxSleep 1000
MinSleep 500

What else can i do?

![image](https://user-images.githubusercontent.com/76137632/102805508-9accf200-43bb-11eb-8f44-2bd6e3b763bc.png)

**Comment by crtoss** [comment, marked as answer]
Did you put the duration inbetween quotes?

Should look like this:

PAGE_SLEEP_MIN="1000"
PAGE_SLEEP_MAX="2000"

Be careful though, most of the retailer sites will give you an error if you update too often.

**Reply by MastermindFIL** [reply]
My stock wont even update for some reason can anyone help?

Im not sure if there is something wrong with my file - 
![streetmerchant 12_31_2020 8_14_42 PM](https://user-images.githubusercontent.com/69785991/103433431-9b2c7080-4ba5-11eb-8373-228e52dc9dd6.png)

**Reply by MastermindFIL** [reply]
nvm fixed

**Reply by csoyuncu** [reply]
what is solve?

**Reply by nsar1907** [reply]
yes whats it (:

**Reply by trillox9** [reply]
> nvm fixed

What is solution? Thanks
